code/game/machinery/computer/aifixer.dm

```diff
@@ -70,34 +71,39 @@
 	popup.open()
 	return
 
+/obj/machinery/computer/aifixer/proc/Fix()

```

Comment:
I can see arguments for something other than process wanting to call this in the future